damus/Views/PostView.swift

```diff
@@ -72,6 +72,7 @@ struct PostView: View {
     @StateObject var tagModel: TagModel = TagModel()
     
     @State private var current_placeholder_index = 0
+    @State private var uploadTask: Task<Void, Never>?
```

Comment:
Multiple upload tasks can be started concurrently. Using a single `Task` variable causes the `cancel` to miss one of the uploads if `uploadTask` is written to more than once (e.g. selecting media from library and then pasting and image).

We should use a list, a set, or some data structure that supports multiple upload tasks

Reply:
Thanks for your feedback and the testing performed with simultaneous uploading case, @danieldaquino . I have added property `uploadTasks' for the purpose of appending the upload tasks in it in same or different uploading combinations. It will be used for canceling all the ongoing tasks. I have tested the change and it works as expected. (also added an empty line after the commit title). thanks